src/content/config.ts

```diff
@@ -0,0 +1,13 @@
+// Astro expects a content/config.ts file, therefore we export the collections here.
+import PagePartialCollection from './PagePartials';
+
+export const collectionMap = {
+  ...PagePartialCollection,
+} as const;
+
+export const collections = Object.entries(collectionMap).reduce<
```

Comment:
Why was it again that we do:
```ts
export default {
  [name]: {
    name,
    collection,
    ...
```
Instead of:
```ts
export default {
  name,
  collection,
  ...
```

It seems this file would be more readable if this could simply be:
```ts
export const allCollections = [
  PagePartialCollection,
] as const;

export const collections = allCollections.reduce((acc, { name, collection }) => {
  acc[name] = collection;
  return acc;
}, {});
```

Is it only because of `@lib/content/index.ts`? Would we consider using `.find()` there instead?